renku/cli/dataset.py

```diff
@@ -65,31 +65,34 @@
 
     $ renku dataset add my-dataset git+ssh://host.io/namespace/project.git
 
-Sometimes you want to import just a specific path within the parent project.
-In this case, use the ``--target`` flag:
+Sometimes you want to import just specific paths within the parent project.
```

Comment:
it looks like some other changes snuck in here - could you rebase on master and remove this commit?

Reply:
I've already did that. I don't get what's wrong!